staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go

```diff
@@ -2350,8 +2350,11 @@ func (i *IngressDescriber) Describe(namespace, name string, describerSettings de
 }
 
 func (i *IngressDescriber) describeBackend(ns string, backend *networkingv1beta1.IngressBackend) string {
-	endpoints, _ := i.CoreV1().Endpoints(ns).Get(context.TODO(), backend.ServiceName, metav1.GetOptions{})
-	service, _ := i.CoreV1().Services(ns).Get(context.TODO(), backend.ServiceName, metav1.GetOptions{})
+	endpoints, errEndpoints := i.CoreV1().Endpoints(ns).Get(context.TODO(), backend.ServiceName, metav1.GetOptions{})
+	service, errService := i.CoreV1().Services(ns).Get(context.TODO(), backend.ServiceName, metav1.GetOptions{})
+	if errEndpoints != nil || errService != nil {
+		return "<none>"
```

Comment:
this is a surprising way of doing things. What about doing it the usual way? And maybe indicate in the string that there was an error?

Usual way (while discarding the error string):
```
endpoints, err := i.CoreV1().Endpoints(ns).Get(context.TODO(), backend.ServiceName, metav1.GetOptions{})
if err != nil {
  return "<none>"
}
service, err := i.CoreV1().Services(ns).Get(context.TODO(), backend.ServiceName, metav1.GetOptions{})
if err != nil {
  return "<none>"
}
```

Reply:
Thanks @apelisee ! I change the err handle process like you said. But about the error string, if we want to show it, we have to change this function to return `(string , error)`. But the origin change for this err handling is to pass the test since the test case doesn't have backend, so I don't know if it is neccessary to change the function.

Reply:
No, I meant we coud the following, using endpoints as an example:
```
endpoints, err := i.CoreV1().Endpoints(ns).Get(context.TODO(), backend.ServiceName, metav1.GetOptions{})
if err != nil {
  return fmt.Sprintf("<none> (failed to fetch endpoints: %v)", err)
}
```

What do you think? Otherwise, it can be misleading ...

Reply:
That's great, I agree.